windows/src/README.md

```diff
@@ -93,13 +93,12 @@ Configure Visual Studio to use two-space tab stops:
 
 ### Delphi setup requirements
 
-Delphi Starter Edition is free and can be downloaded [here](https://www.embarcadero.com/products/delphi/starter).
-Some features of Keyman are [limited](#delphi_starter---building-with-delphi-starter-edition) when built using
-the Starter Edition. You must set the `DELPHI_STARTER` environment variable when using the Starter Edition.
+Delphi Community Edition is free and can be downloaded [here](https://www.embarcadero.com/products/delphi/????).
```

Comment:
This link worked for me
https://www.embarcadero.com/products/delphi/starter/free-download

Reply:
Ooops forgot to fix that link when I got off the plane... :)